projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/ui/Messages.kt

```diff
@@ -41,7 +42,17 @@ internal fun showMessage(project: Project, header: String, message: String): Boo
 
   if (widget is ProjectorStatusWidget) {
     val gotItMessage = GotItMessage.createMessage(header, message).setDisposable(widget)
-    gotItMessage.show(RelativePoint.getCenterOf(widget.component), Balloon.Position.above)
+
+    with(widget.component) {
+      gotItMessage.show(
+        object : PositionTracker<Balloon?>(this) {
+          override fun recalculateLocation(baloon: Balloon): RelativePoint {
+            return RelativePoint.getCenterOf(this@with)
+          }
+        },
+        Balloon.Position.above)
+    }
```

Comment:
Minor: looks like now we can improve it even more ;)

```suggestion
    gotItMessage.show(
      object : PositionTracker<Balloon?>(widget.component) {
        override fun recalculateLocation(baloon: Balloon) = RelativePoint.getCenterOf(widget.component)
      },
      Balloon.Position.above,
    )
```

@ARTI1208, please take a look, test it, and take the final decision. I'm going out of this PR.